src/webgpu/api/validation/encoding/beginRenderPass.spec.ts

```diff
@@ -163,8 +163,9 @@ g.test('occlusion_query_set,device_mismatch')
     });
     t.trackForCleanup(occlusionQuerySet);
 
-    const encoder = t.createEncoder('render pass', { occlusionQuerySet });
-    encoder.validateFinish(!mismatched);
+    t.expectValidationError(() => {
+      t.createEncoder('render pass', { occlusionQuerySet });
```

Comment:
Internally this will make multiple WebGPU calls so I think it should be replaced with direct calls to createCommandEncoder and beginRenderPass